README.md

```diff
@@ -4,7 +4,20 @@ Built on top of the [Wasmer](https://github.com/wasmerio/wasmer) runtime.
 
 ## Setup
 
-Run `dart run wasm:setup` to build the Wasmer runtime.
+1. Start by installing the tools needed to build the Wasmer runtime:
+
+   * Install the [Rust SDK].
+
+   * On Windows, to get `link.exe`, install the [Visual Studio build tools]
+     with the "Desktop development with C++"-option selected.
+
+1. Next run `dart run wasm:setup` to build the Wasmer runtime.
+
+1. Finally, install [LLVM] to enable compiling C code to WASM with `clang`
```

Comment:
Hmmm, might need to reword this a bit. LLVM is just one way of getting wasm files, and the way this is structured implies that it's a necessary step (since the other 2 points are necessary).

Maybe mark it optional, or reword it like "install a compiler that can produce wasm code, such as LLVM", or maybe just remove it and just link to it below like we had it before?

Reply:
Ah, I see. I removed this step and kept the LLVM link below.